packages/o/openblas/xmake.lua

```diff
@@ -56,11 +56,10 @@ package("openblas")
         add_frameworks("Accelerate")
     end
     on_load("macosx", "linux", "mingw@windows,msys", function (package)
-        if package:config("fortran") then
-            package:add("deps", "gfortran", {system = true})
-        end
         if package:config("openmp") then
             package:add("deps", "openmp")
+        else
```

Comment:
we should use package:config("fortran")

Reply:
Sorry, I don't get it. Do you mean `package:config("gfortran")`? I don't see any config options in `gfortran/xmake.lua`.

Reply:
>         add_configs("fortran", {description = "Compile with fortran enabled.", default = is_plat("linux"), type = "boolean"})

openblas's configs